pkg/liquidity-mining/contracts/gauges/gnosis/GnosisRootGauge.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+import "../StakelessGauge.sol";
+
+interface IGnosisBridge {
+    function relayTokens(
+        IERC20 token,
+        address _receiver,
+        uint256 _value
+    ) external;
+}
+
+contract GnosisRootGauge is StakelessGauge {
+    IGnosisBridge private immutable _gnosisBridge;
+
+    // This value is kept in storage and not made immutable to allow for this contract to be proxyable
```

Comment:
```suggestion
    // This value is kept in storage and not made immutable to allow for the recipient to be set during `initialize`
    // and not in the constructor, enabling this contract to be a proxy.
```

I'd rephrase a bit. Maybe the suggestion is too verbose, but I had to read the comment a couple of times to understand to understand the meaning of `proxyable` in this context :)